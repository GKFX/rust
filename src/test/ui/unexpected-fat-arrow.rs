fn main() {
    // JavaScript-style arrow function.
    let _lambda = (a, b) => { println!(); a + b };
//~^ ERROR expected one of `.`, `;`, `?`, `else`, or an operator, found `=>`
}
