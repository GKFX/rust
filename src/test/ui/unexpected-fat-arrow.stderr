error: expected one of `.`, `;`, `?`, `else`, or an operator, found `=>`
  --> $DIR/unexpected-fat-arrow.rs:3:26
   |
LL |     let _lambda = (a, b) => { println!(); a + b };
   |                          ^^ expected one of `.`, `;`, `?`, `else`, or an operator
   |
   = help: closures are written `|a, b| a + b` and greater-than-or-equal is `>=`.

error: aborting due to previous error

